engine/access/rpc/backend/backend_scripts.go

```diff
@@ -14,12 +16,16 @@ import (
 	"github.com/onflow/flow-go/storage"
 )
 
+// uniqueScriptLoggingTimeWindow is the duration for checking the uniqueness of scripts sent for execution
+const uniqueScriptLoggingTimeWindow = time.Duration(10) * time.Minute
```

Comment:
Nit: the compiler will figure out the type in this case
```suggestion
const uniqueScriptLoggingTimeWindow = 10 * time.Minute
```